ScrollableSegmentedControl/ScrollableSegmentedControl.swift

```diff
@@ -481,6 +488,8 @@ public enum ScrollableSegmentedControlSegmentStyle: Int {
         static let defaultFont = UIFont.systemFont(ofSize: 14)
         static let defaultTextColor = UIColor.darkGray
         
+        var underlineHeight: CGFloat = 3.0
```

Comment:
Perhaps make this just proxy getter and setter for  (_underlineHeightConstraint constant mentioned below). Setter would most likely need to invalidate layout.